### PR TITLE
Nazrhom/cli platform interface

### LINKED ISSUE #328: CLI support for bidder approval using Platform server
- [ ] Publish bidder approval on Platform server (new CLI action for seller?)
- [ ] Query for bidder approval and use it in `newBid`

### PR DESCRIPTION
**Pull request by nazrhom**

### CODE REVIEW

**Comment by uhbif19** [comment]
@nazrhom Does it close #328 ?

File: app/CLI/Actions.hs [hunk 1]
```diff
@@ -227,21 +229,42 @@ handleCliAction sendRequestToDelegate currentDelegateStateRef userAction = do
...
-              L2 -> do
-                (_, _, bidderSigningKey) <- addressAndKeys
-                let bidDatum =
-                      createStandingBidDatum terms bidAmount sellerSignature bidderSigningKey
-                liftIO $
-                  sendRequestToDelegate $
-                    DelegateInterface.NewBid
-                      { auctionTerms = terms
-                      , datum = bidDatum
+          approval <-
```
Comment (uhbif19) [inline app/CLI/Actions.hs:232]: This should be separated.

**Comment by nazrhom** [comment]
> @nazrhom Does it close #328 ?

yeah, added to linked issues

File: app/CLI/Actions.hs [hunk 2]
```diff
@@ -228,7 +230,18 @@ handleCliAction sendRequestToDelegate currentDelegateStateRef userAction = do
...
+          r <-
+            liftIO $
+              sendRequestToPlatform $
+                MkSome BidderApproval $
+                  Query $
+                    MkQuery
+                      { filters = [ByApprovedBidder $ Eq actor]
+                      , limit = Nothing
+                      }
+          liftIO $ print r
```
Comment (uhbif19) [inline app/CLI/Actions.hs]:
```
```

**Pull request closed without merging**